nupic/research/frameworks/dendrites/metrics/metrics.py

```diff
@@ -313,3 +314,126 @@ def entropy(x):
     _max_entropy = math.log(num_segments)
 
     return _entropy, _max_entropy
+
+
+def repr_overlap_matrix(activations, targets):
```

Comment:
Typically, we try and avoid shorthands for naming to be explicit when we can be. For local variables and arguments, I think it's often more acceptable to use shorthands/abbreviations since the context should be clear enough. Of course, we have no hard and fast rules.

If `repr` had a standard ML interpretation (like `conv`), your original would be a good name. But given our tendency to use full names when possible, and the fact that `repr` is a builtin python function, I'd vote for calling it `representaion_overlap_matrix`.

Reply:
Fixed.